pkgs/by-name/wa/wasmi/package.nix

```
@@ -2,27 +2,24 @@
   lib,
   rustPlatform,
   fetchFromGitHub,
+  nix-update-script,
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "wasmi";
-  version = "0.31.0";
+  version = "0.40.0";
 
   src = fetchFromGitHub {
     owner = "paritytech";
     repo = "wasmi";
     rev = "v${version}";
```

we could even update this while we're at it

```suggestion
    tag = "v${version}";
```